sampler/README.md

```diff
@@ -128,18 +130,18 @@ points distributed across different stats (e.g., +2 wisdom and +3
 intelligence) or concentrated within a single stat (e.g., +5
 charisma).
 
-How would we randomly sample these stats? The easiest way is probably
-to first sample the overall item bonus. Then, we sample the way the
+How would we randomly sample this distribution? The easiest way is probably
```

Comment:
I would say "sample from" rather than just "sample" but otherwise this change is fine.